lib/monitoring/monitoringClientInterface.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const assert = require('assert')
+const client = require('prom-client');
+
+/**
+ * Wrapper for Prometheus client - https://github.com/siimon/prom-client
+ */
+
+var references = {};
+var namespace = '';
```

Comment:
I don't see why you'd ever want/need to 'namespace' metrics. All filtering can/should be done based on Prometheus' dynamic labels (added by the scraper).

Even to the contrary: you *don't* want to 'namespace' metrics because then you're certainly not following the de-facto 'standard' metric names used by various services and exporter.

Reply:
removed namespace.